src/valid/type.rs

```diff
@@ -350,7 +356,7 @@ impl super::Validator {
                             return Err(TypeError::NonPositiveArrayLength(const_handle));
                         }
 
-                        TypeFlags::SIZED
+                        TypeFlags::SIZED | TypeFlags::ARGUMENT
```

Comment:
is this true that anything `SIZED` can be passed as an argument?

Reply:
Now that I mention it. I'm not sure but I think the arrays with constant size that can't be passed aren't themselves valid types, but I'm not sure I'll have to look into it.

Reply:
Actually the old code accepts any sized array as an argument